network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/share-buttons.html

```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+
+{% with request.scheme|add:'://'|add:request.get_host|add:request.get_full_path as current_url %}
```

Comment:
let's stick with double quotes if we can. But also: do we need this? Pages have a built in `page.full_url` to get the fully qualified URL so can we not just pass the page itself to this fragment by using `{% include "./fragments/share-buttons.html" with page=page ...... %}`?